app/models/project/project.js

```diff
@@ -6,6 +6,9 @@ module.exports = (sequelize, Sq) => {
     name: {
       type: Sq.STRING,
       allowNull: false,
+      jsonSchema: {
+        schema: {type: 'string', minLength: 2},
```

Comment:
is there any particular reason for this minlength?

Reply:
It's just to block the creation of empty or 1 character project names.